exonum/src/helpers/mod.rs

```diff
@@ -156,6 +162,29 @@ pub fn byzantine_quorum(total: usize) -> usize {
     total * 2 / 3 + 1
 }
 
+/// Creates and initializes RustRuntime and GenesisConfig with information from collection of InstanceCollection.
+pub fn create_rust_runtime_and_genesis_config(
```

Comment:
Not sure about this combination. I think that genesis config and rust runtime with its instances should be created independently.

Reply:
I also don't like it much either, but for the moment entities of different kind (service factory and wrapper of `InstanceSpec`) comes from single source (InstanceCollection).

Reply:
I believe it's one more argument for removing the `InstanceCollection` abstraction then.

Reply:
Mentioned nit finished tasks in TODOs and in ECR-3913.